src/librustdoc/html/render/span_map.rs

```diff
@@ -73,6 +74,22 @@ impl<'tcx> SpanMapVisitor<'tcx> {
                 Some(def_id)
             }
             Res::Local(_) => None,
+            Res::PrimTy(p) => {
```

Comment:
Note that not all primitives show up in PrimTy, it can only represent "path-like" primitives (so not e.g. arrays and tuples). Is that intentional?
https://doc.rust-lang.org/nightly/nightly-rustc/rustc_hir/hir/enum.PrimTy.html

Reply:
Yes it is. However a FIXME would be nice, I'll add one.